app/models/druid_version_zip.rb

```diff
@@ -30,6 +30,16 @@ def ensure_zip_directory!
     Pathname.new(file_path).tap { |pn| pn.dirname.mkpath }
   end
 
+  # @param [Integer] count Number of parts
+  # @return [Array<String>] Ordered pathnames expected to correspond to a zip broken into a given number of parts
+  def expected_parts(count = 1)
```

Comment:
i'm thinking "expected_part_names" might be more clear ...

Reply:
Let me put it this way ... I was confused when I first looked at this method;  was it going to give me the expected pieces?  The number of expected parts?  oooOOOOoooooh - the NAMES of the parts.  I'm coming at this from a different frame of reference, and I thought the name was too vague and contributed to my confusion.

Reply:
- The expected pieces? Yes?
- The number of parts? Yes, indirectly, but also tautologically, because the number of parts is the documented `@param`.  

The constraint is that this logic has to operate without being able to rely on the file glob on zip transfers directory, which may no longer be comprehensive (or even have **any** matching zip files).  It is precisely symmetric to `parts` which actually does that glob, used immediately after successful zip creation.  The name reflects the relationship between `expected_parts` and `parts` and any drift obscures that.  

The return value is:
```
Ordered pathnames expected to correspond to a zip broken into a given number of parts
```
I think that is pretty descriptive and enough to relieve any confusion.  If it would be helpful, I would rename it `parts_expected`, which would move it down to live next to its counterpart.